src/test/java/com/epam/tc/hw2/ex2/ExerciseSecondTest.java

```diff
@@ -0,0 +1,60 @@
+package com.epam.tc.hw2.ex2;
+
+import com.epam.tc.hw2.BrowserTestBase;
+import java.util.List;
+import org.assertj.core.api.SoftAssertions;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.Select;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.Test;
+
+
+public class ExerciseSecondTest extends BrowserTestBase {
+
+    @Test
+    public void testDifferentElements() {
+        // 5. Open through the header menu Service -> Different Elements Page - Page is opened
+        driver.findElement(By.className("menu-title")).click();
+
+        new WebDriverWait(driver, 10)
+                .until(ExpectedConditions.elementToBeClickable(By.linkText("Different elements")));
+        driver.findElement(By.linkText("Different elements")).click();
+        SoftAssertions softAssertions = new SoftAssertions();
+        softAssertions.assertThat(driver.getTitle()).as("check the title").isEqualTo("Different Elements");
+
+        // 6. Select checkboxes (Water and Wind)
+        driver.findElement(By.xpath("//*[text()[contains(., 'Water')]]")).click();
+        softAssertions.assertThat(driver.findElement(By.xpath("//*[text()[contains(., 'Water')]]"))
+                .isEnabled()).as("check the checkbox Water is checked").isTrue();
```

Comment:
Why do you check that checkbox is enabled after clicking on it? Moreover, you only need to click on it according to the task.